connectors/rsk.go

```diff
@@ -30,7 +30,7 @@ const (
 	rpcSleep   time.Duration = 2 * time.Second
 	rpcTimeout time.Duration = 5 * time.Second
 	ethSleep   time.Duration = 5 * time.Second
-	ethTimeout time.Duration = 5 * time.Minute
+	EthTimeout time.Duration = 5 * time.Minute
```

Comment:
I am not sure about making this public since the idea of taking a context in getStatus is to be able to define a timeout outside of the connectos package. Using this value is conceptually the same as not taking a context as a parameter.

Reply:
reverted as it's not necessary anymore.